lib/db/repositories/ReferralRepository.ts

```diff
@@ -112,23 +117,56 @@ class ReferralRepository {
   private static sanityCheckConfig = (
     config: ReferralConfig | null | undefined,
   ) => {
+    const MAX_PREMIUM_PERCENTAGE = 100;
+    const MIN_PREMIUM_PERCENTAGE = -100;
+    const MIN_EXPIRATION = 120;
+    const MAX_EXPIRATION = 60 * 60 * 24;
+    const MAX_ROUTING_FEE = 0.005;
+
     const sanityCheckPairConfig = (cfg: ReferralPairConfig) => {
-      if (cfg.maxRoutingFee) {
-        if (cfg.maxRoutingFee < 0 || cfg.maxRoutingFee > 0.005) {
-          throw 'maxRoutingFee out of range';
-        }
+      if (
+        cfg.maxRoutingFee &&
```

Comment:
```suggestion
        cfg.maxRoutingFee !== undefined &&
```

JavaScript being JavaScript. I missed that